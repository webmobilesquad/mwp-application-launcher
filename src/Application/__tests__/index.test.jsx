import render from 'preact-render-spy';
import { Application } from '../index';

describe('<Application />', () => {
  test('<Application /> matches snapshot', () => {
    const application = {
      name: 'Test application',
      url: 'test/url',
      icon: 'icon.png',
    };
    const currentUrl = '/current/url';
    expect(render(<Application application={application} url={currentUrl} />)).toMatchSnapshot();
  });
});
