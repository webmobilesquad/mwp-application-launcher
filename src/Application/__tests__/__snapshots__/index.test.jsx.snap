// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Application /> <Application /> matches snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<li class="Application">
  <a
    href="test/url"
    title="Test application"
  >
    <img
      src="icon.png"
      alt="Test application"
     />
  </a>
</li>

`;
